tests/containers/pydantic_schemas/openshift/README.md

```diff
@@ -0,0 +1,20 @@
+Openshift schemas as [Pydantic](https://pydantic-docs.helpmanual.io/) datamodels
+
+Generated with [koxudaxi/datamodel-code-generator](https://github.com/koxudaxi/datamodel-code-generator)
+from [Kubernetes's OpenAPI Specification](https://github.com/kubernetes/kubernetes/tree/master/api/openapi-spec)
+
+Openshift does not seem to be publishing their API anymore
+([openshift/origin#25643](https://github.com/openshift/origin/issues/25643)), so fetch from live cluster.
+
+```shell
+$ oc get --raw /openapi/v2 > openapi-v2.json
+```
+
+Expect the generation to take long time (minutes)
```

Comment:
When and how often is this supposed to be run? Do we want to cache it somewhere to avoid waiting minutes every time? :thinking:

Reply:
I don't intend to run anything like this, actually. I committed this README.md by mistake, I only intended to push changes to that _other_ README.md.

I got annoyed that kubernetes-py has very inadequate type hints and figured it will be better to run our own generator, something like odh-tests did. But I decided not to look into it for now.

Reply:
i've removed the README and the directory. I don't intend to go this route, at least not yet.